README.md

```diff
@@ -285,6 +287,16 @@ Additional querying methods provided, by endpoint:
 
 For security reasons, methods like `.revisions()` and `.users()` require the request to be authenticated.
 
+#### toString()
+
+To get the URI of the resource _without_ making a request, call `.toString()` at the end of a query chain:
+
+```js
+var uriString = wp.posts().id( 7 ).embed().toString();
```

Comment:
`String(wp.posts())` looks pretty cool also ;)  but yeah - your way to document it is pretty great